src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserPostDao.java

```diff
@@ -0,0 +1,55 @@
+package com.vinylteam.vinyl.dao.jdbc;
+
+import com.vinylteam.vinyl.dao.UserPostDao;
+import com.vinylteam.vinyl.dao.jdbc.mapper.UserPostRowMapper;
+import com.vinylteam.vinyl.entity.UserPost;
+import com.zaxxer.hikari.HikariDataSource;
+import org.postgresql.util.PSQLException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class JdbcUserPostDao implements UserPostDao {
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private final String INSERT = "INSERT INTO public.user_posts" +
+            " (user_id, name, email, theme, message)" +
+            " VALUES (?, ?, ?, ?, ?)";
```

Comment:
I think we need also field for Date & Time when message was wrote

Reply:
> I think we need also field for Date & Time when message was wrote

I allso think about it, but there is no such field in basic DB schema

Reply:
> I think we need also field for Date & Time when message was wrote

We will add in in future in separate task